osmaxx/excerptexport/migrations/0001_squashed_0016_extractionorder_process_start_time.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-05-23 11:42
+from __future__ import unicode_literals
+
+from django.conf import settings
+import django.contrib.gis.db.models.fields
+from django.core.files.storage import FileSystemStorage
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class PrivateFileSystemStorage(FileSystemStorage):
+    # This is a small hack to still be able to use the standard FileSystemStorage
+    # without having the hardcoded PRIVATE_MEDIA_ROOT in automatic generated migrations.
```

Comment:
Is this comment still applicable now that the class's code has been moved into a migration script?